packages/angular/cli/lib/config/workspace-schema.json

```diff
@@ -106,6 +106,15 @@
               "type": "string"
             }
           }
+        },
+        "completion": {
+          "description": "Enable shell command completions.",
```

Comment:
Nit: can you improve the descriptions?

- completion description: Angular CLI completion settings.
- prompted description: Whether the user has been prompted to add completion command prompt.